ddl/schema.go

```diff
@@ -77,7 +77,7 @@ func (d *ddl) onDropSchema(t *meta.Meta, job *model.Job) (ver int64, _ error) {
 	}
 	if dbInfo == nil {
 		job.State = model.JobStateCancelled
-		return ver, infoschema.ErrDatabaseDropExists.GenByArgs("")
+		return ver, infoschema.ErrDatabaseDropExists.GenByArgs(job.SchemaName)
```

Comment:
Could we use `meta.GetDatabase(dbID)` to get the schema name?
Then we can remove the field of `SchemaName` in `Job`.

Reply:
And we can update error messages in `getTableInfo` too.

Reply:
We can talk about that.

Reply:
If we get schema name failed by `meta.GetDatabase(dbID)`, maybe we can use `schema ID`. Because we encounter this case only when we execute to drop the same schema in parallel. 
The problem we can talk about.